pkg/storage/chunk/cache/memcached.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	ErrMemcachedClosed = errors.New("memcached is closed")
+	ErrMemcachedClosedByClient = errors.New("memcached is closed by client")
```

Comment:
This doesn't make sense; do you mean a memcached _connection_ is closed by the client?
```suggestion
	ErrMemcachedConnClosed = errors.New("memcached connection is closed")
```

Reply:
I don't want to confuse this with `connection` here. Because we just closing the `Memcached` struct created by client. So that it can no longer be used for storing or fetching.

May be `memcached` word is confusing. Going to change to `cache is stopped by client` instead. The actual message is client called `Stop()` method deliberately.

Reply:
Makes sense 👍